skgrid/set_model_params.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+import os
+import sys
+import yaml
+import json
+import argparse
+from sklearn.model_selection import ParameterGrid
+from sklearn.ensemble import AdaBoostClassifier
+from sklearn.naive_bayes import BernoulliNB
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import ExtraTreesClassifier
+from sklearn.naive_bayes import GaussianNB
+from sklearn.gaussian_process import GaussianProcessClassifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.neural_network import MLPClassifier
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.linear_model import PassiveAggressiveClassifier
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import SGDClassifier
+from sklearn.svm import SVC
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--config", help='path to grid config yaml file')
+parser.add_argument("--cancer", help='cancer abbrev (ex. BRCA)')
+parser.add_argument("--platform", help='data platform abbrev (ex. GEXP)')
+parser.add_argument("-n", default=1, type=int)
+args = parser.parse_args()
+
+n = args.n
+
+def chunks(lst, n):
+    for i in range(0, len(lst), n):
+        yield lst[i:i + n]
+
+classifers = {
+    "sklearn.ensemble.AdaBoostClassifier": AdaBoostClassifier,
+    "sklearn.naive_bayes.BernoulliNB": BernoulliNB,
+    "sklearn.tree.DecisionTreeClassifier": DecisionTreeClassifier,
+    "sklearn.ensemble.ExtraTreesClassifier": ExtraTreesClassifier,
+    "sklearn.naive_bayes.GaussianNB": GaussianNB,
+    "sklearn.gaussian_process.GaussianProcessClassifier": GaussianProcessClassifier,
+    "sklearn.neighbors.KNeighborsClassifier": KNeighborsClassifier,
+    "sklearn.linear_model.LogisticRegression": LogisticRegression,
+    "sklearn.neural_network.MLPClassifier": MLPClassifier,
+    "sklearn.naive_bayes.MultinomialNB": MultinomialNB,
+    "sklearn.linear_model.PassiveAggressiveClassifier": PassiveAggressiveClassifier,
+    "sklearn.ensemble.RandomForestClassifier": RandomForestClassifier,
+    "sklearn.linear_model.SGDClassifier": SGDClassifier,
+    "sklearn.svm.SVC": SVC
+}
+
+# Load best model yaml
+with open('/skgrid/select_model.yml', 'r') as handle:
```

Comment:
Rather then having the path hard coded here, add it as an argument to the command line parser what that default value. That way it will operate the same if no value is provided, but could be overridden if needed.

Reply:
added as an argument to command line parser with default value in commit [57c2270](https://github.com/ohsu-comp-bio/gdan-tmp-models/pull/7/commits/57c227044e43432e9e415ad46894c50f4f5bb282)